<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UtilGuiClass</class>
 <widget class="QMainWindow" name="UtilGuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UtilGui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1</y>
      <width>591</width>
      <height>320</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="m_pBtChineseSupport">
         <property name="text">
          <string>中文支持</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_pLbPath">
         <property name="text">
          <string>路径</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="m_pBtCompare">
         <property name="text">
          <string>字符比较</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pBtHexString">
         <property name="text">
          <string>16进制和字符串</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pBtSerialPort">
         <property name="text">
          <string>串口调试</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pBtCloseSerialPort">
         <property name="text">
          <string>关闭串口</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="m_pBtJoyStick">
       <property name="text">
        <string>游戏手柄</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pBtOpencvDemo">
       <property name="text">
        <string>OpenCV功能测试-使用2.4.11</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="m_pBtSqlite">
         <property name="text">
          <string>测试sqlite</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pBtJson2Db">
         <property name="text">
          <string>Json中文压入数据库</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="m_pBtZipFile">
         <property name="text">
          <string>压缩文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pBtUnzipFile">
         <property name="text">
          <string>解压文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pBtStartQueue">
         <property name="text">
          <string>启动队列</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pBtStopQueue">
         <property name="text">
          <string>测试结构体压入队列</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pBtQueueSize">
         <property name="text">
          <string>查看队列大小</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="m_pBtPrint">
       <property name="text">
        <string>打印</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pBtWeb">
       <property name="text">
        <string>打开QtWeb</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="m_pBtStartThread">
         <property name="text">
          <string>开启线程</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pBtStopThread">
         <property name="text">
          <string>停止线程</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>593</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="UtilGui.qrc"/>
 </resources>
 <connections/>
</ui>
